recipes/chemfiles/meta.yaml

```diff
@@ -0,0 +1,28 @@
+{% set version = "0.6.0" %}
+
+package:
+  name: chemfiles
+  version: {{ version }}
+
+build:
+  number: 0
```

Comment:
Please add `skip: true  # [win]`. That should fix the CI failure. Also feel free to add `[skip appveyor]` in your commit message to avoid having to run that CI as it is not used.